skiko/src/awtTest/kotlin/org/jetbrains/skiko/SkiaLayerPerformanceTest.kt

```diff
@@ -179,6 +188,7 @@ class SkiaLayerPerformanceTest {
     }
 
     @Test
+    @Ignore
```

Comment:
- why we ignore only one test? the test below is more heavy, why it pass?
- it it also fails, it is better to just to pass `skiko.test.performance.enabled`, because it is often needed to run these tests manually